app/src/main/kotlin/com/wire/android/util/CamelCase.kt

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
What do you think of having this file named after `StringsExt` or `StringsUtils`. 
We could put other strings related manipulation here :D